amplitude_python_sdk/v2/client.py

```diff
@@ -81,3 +89,75 @@ def batch_upload_events(
         """  # pylint: disable=line-too-long
         req_data = self._construct_event_request_data(events, options)
         return self._log_events_internal(routes.BATCH_API, req_data, timeout)
+
+    @staticmethod
+    def _construct_chart_request_data(
+        annotation: ChartAnnotationsV2
+    ) -> Dict[str, Any]:
+        """
+        Convert ChartAnnotationsV2 model to dictionary payload for _chart_annotations_post request
+        """
+        req_data = annotation.dict(exclude_none=True)
+
+        req_data["date"] = req_data["date"].__str__()
+
+        return req_data
+
+    def _charts_annotations_post(
+        self, req_data: Dict[str, Any], timeout: int = 5,
+    ) -> requests.Response:
+        """
+        Create an annotation
+        """
+        try:
+            resp = requests.post(
+                url=self.api_endpoint_2 + routes.CHART_ANNOTATIONS_API,
+                auth=(self.api_key, self.secret_key),
+                json=req_data,
```

Comment:
I think here we can just pass the JSON string directly and change this argument to `data=`, see [here](https://2.python-requests.org/en/master/user/quickstart/#more-complicated-post-requests) for more explanation.

Reply:
Done